screens/Home/MyVcsTab.tsx

```diff
@@ -374,12 +383,6 @@ export const MyVcsTab: React.FC<HomeScreenTabProps> = props => {
         </Row>
       </MessageOverlay>
 
-      <ErrorMessageOverlay
-        translationPath={'MyVcsTab'}
-        isVisible={controller.isSavingFailedInIdle}
-        error={storeErrorTranslationPath}
```

Comment:
Please remove 'storeErrorTranslationPath' constant as it is no more getting used

Reply:
removed

Reply:
Removed